docs/getting-started/android/screenshots.md

```diff
@@ -1,7 +1,122 @@
+
+## Installing _screengrab_
+
+Install the gem:
+
+```
+sudo gem install screengrab
+```
+
+### Gradle dependency
+
+Add the test dependency to your Gradle build:
+
+```java
+androidTestCompile 'tools.fastlane:screengrab:0.3.2'
+```
+
+### Configuring your Manifest Permissions
+Ensure that the following permissions exist in your `src/debug/AndroidManifest.xml`
+
+```xml
+<!-- Allows unlocking your device and activating its screen so UI tests can succeed -->
+<uses-permission android:name="android.permission.DISABLE_KEYGUARD"/>
+<uses-permission android:name="android.permission.WAKE_LOCK"/>
+
+<!-- Allows for storing and retrieving screenshots -->
+<uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
+<uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
+
+<!-- Allows changing locales -->
+<uses-permission android:name="android.permission.CHANGE_CONFIGURATION" />
+```
+
+## Configuring your UI Tests for _screengrab_
+
+1.  Add `@ClassRule public static final LocaleTestRule localeTestRule = new LocaleTestRule();` to your tests class to handle automatic switching of locales
+2.  To capture screenshots, add the following to your tests `Screengrab.screenshot("name_of_screenshot_here");` on the appropriate screens
+
+## Generating Screenshots with Screengrab
+
+- Run `./gradlew assembleDebug assembleAndroidTest` to generate debug and test APKs
+- Run `fastlane screengrab` in your app project directory to generate screenshots
+    - You will be prompted to provide any required parameters which are not in your `Screengrabfile`, or provided as command line arguments
+- Your screenshots will be saved to `fastlane/metadata/android` in the directory where you ran `fastlane screengrab`
+
+### Improved screenshot capture with UI Automator
+
+As of _screengrab_ 0.5.0, you can specify different strategies to control the way _screengrab_ captures screenshots. The newer strategy delegates to [UI Automator](https://developer.android.com/topic/libraries/testing-support-library/index.html#UIAutomator) which fixes a number of problems compared to the original strategy:
+
+* Shadows/elevation are correctly captured for Material UI
+* Multi-window situations are correctly captured (dialogs, etc.)
+* Works on Android N
+
+However, UI Automator requires a device with **API level >= 18**, so it is not yet the default strategy. To enable it for all screenshots by default, make the following call before your tests run:
+
+```java
+Screengrab.setDefaultScreenshotStrategy(new UiAutomatorScreenshotStrategy());
+```
+
+# Upload Screenshots to Google Play
+
+After generating your screenshots using `fastlane screengrab`, you'll usually want to upload them to Google Play.
+
+**TODO: Bring in _supply_ getting started content**
+
+To upload the screenshots stored in `fastlane/screenshots`, just run
```

Comment:
The screenshots are stored in `fastlane/metadata/android` not fastlane/screenshots